build/alpine/Dockerfile

```diff
@@ -1,32 +0,0 @@
-# Requires docker >= 17.05 (requires support for multi-stage builds)
```

Comment:
what was the purpose of this file before?  i can't find a reason why it was there in the code you changed / removed.